java/arcs/android/storage/database/DatabaseImpl.kt

```diff
@@ -1023,6 +1023,7 @@ class DatabaseImpl(
                     GROUP BY storage_key_id, storage_key, orphan
                     HAVING entities.creation_timestamp < $twoDaysAgo
                     AND (orphan OR noRef)
+                    AND storage_keys.storage_key NOT LIKE 'inline%'
```

Comment:
How will we know when/whether we need to add to this or change it with future type support work? Do we need the checklist of things to consider to help remind ourselves, or some test that asserts the known types we have today exist and there are no others so we'll get a test breakage unless this is considered for new ones, etc (just brainstorming for effect mostly).

Reply:
Yes agree this is a brittle, and string comparison I think will also be unefficient. I just filed b/165391190 to improve on this.